Data/ByteString/Builder/RealFloat.hsc

```diff
@@ -256,17 +266,17 @@ showFixed m e prec =
         digitsToBuilder = fmap (char7 . intToDigit)
 
 #if MIN_VERSION_base(4,6,0)
-dquot10 :: Word# -> Word#
+dquot10 :: Word## -> Word##
```

Comment:
I don't think there's a `Word##` type in GHC. There's `Word#`, `Word32#` and `Word64#`.

Reply:
These are all artifacts of moving to an hsc file where the the preprocessor escapes double-hashes to single-hashes